### PR TITLE
28 palette styling

### PR DESCRIPTION
**Pull request by Nikomus**
Fixes: #28 

Also some general dependency updates

### CODE REVIEW

File: next-env.d.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
 /// <reference types="next/image-types/global" />
 
 // NOTE: This file should not be edited
```
Comment (katjam) [inline next-env.d.ts:4]: lol are you sure you want to do that ---^

**Review by katjam — APPROVED** [review]
Seems OK to me.

File: components/ColourButton.tsx [hunk 2]
```diff
@@ -13,22 +13,28 @@ interface colourButtonProps {
...
   justify-content: space-around;
-  overflow: auto;
+  overflow: hidden;
   aspect-ratio: 1;
-  flex-basis: 25%;
-  margin-bottom: 2px;
+  margin: 5px;
+  min-height: 44px;
+  min-width: 44px;
+  width: calc(20% - 10px);
```
[the remark this answers is not in the document]
Comment (Nikomus) [reply components/ColourButton.tsx:26]: This should be in one row for anything larger than a phone

File: next-env.d.ts (hunk 1 above)
Comment (Nikomus) [reply next-env.d.ts:4]: It wasn't me! Next regenerates this file.

**Comment by katjam** [comment]
Merge away!

**Pull request merged**